### PR TITLE
[aot] Avoid raising exceptions in the AOT runtime.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by vargaz** [comment]
@monojenkins merge

**Comment by vargaz** [comment]
@monojenkins build

File: mono/mini/aot-runtime.c [hunk 1]
```diff
@@ -5825,6 +5831,12 @@ mono_aot_get_method (MonoDomain *domain, MonoMethod *method)
 	return NULL;
 }
 
+gpointer
+mono_aot_get_method_checked (MonoDomain *domain, MonoMethod *method, MonoError *error)
+{
+	return NULL;
```
Comment (lambdageek) [inline mono/mini/aot-runtime.c]: Needs a `mono_error_init(error)` before returning.

File: mono/mini/aot-runtime.c [hunk 2]
```diff
@@ -5873,7 +5885,7 @@ mono_aot_patch_plt_entry (guint8 *code, guint8 *plt_entry, gpointer *got, mgreg_
 }
 
 gpointer
-mono_aot_get_method_from_vt_slot (MonoDomain *domain, MonoVTable *vtable, int slot)
+mono_aot_get_method_from_vt_slot (MonoDomain *domain, MonoVTable *vtable, int slot, MonoError *error)
 {
 	return NULL;
```
Comment (lambdageek) [inline mono/mini/aot-runtime.c]: Needs a `mono_error_init (error)` before returning.

**Pull request merged**